doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -335,6 +335,9 @@ nRF Desktop
     The negative impact of USB polling jitter is more visible in case of USB High-Speed.
   * The Fast Pair sysbuild configurations to align the application with the sysbuild Kconfig changes for controlling the Fast Pair provisioning process.
     The Nordic device models intended for demonstration purposes are now supplied by default in the nRF Desktop Fast Pair configurations.
+  * The :ref:`nrf_desktop_dvfs` to no longer consume the ``ble_peer_conn_params_event``.
```

Comment:
```suggestion
  * The :ref:`nrf_desktop_dvfs` to no longer consume the :c:struct:`ble_peer_conn_params_event`.
```
This might do the link to the event.